analyzers/tests/SonarAnalyzer.UnitTest/Rules/StaticFieldInGenericClassTest.cs

```diff
@@ -27,18 +27,23 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class StaticFieldInGenericClassTest
     {
+        private readonly VerifierBuilder builder = new VerifierBuilder<StaticFieldInGenericClass>();
+
         [TestMethod]
         public void StaticFieldInGenericClass() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\StaticFieldInGenericClass.cs", new StaticFieldInGenericClass());
+            builder.AddPaths("StaticFieldInGenericClass.cs").Verify();
 
 #if NET
+
         [TestMethod]
         public void StaticFieldInGenericClass_CSharp9() =>
-            OldVerifier.VerifyAnalyzerFromCSharp9Library(@"TestCases\StaticFieldInGenericClass.CSharp9.cs", new StaticFieldInGenericClass());
+            builder.AddPaths("StaticFieldInGenericClass.CSharp9.cs").WithOptions(ParseOptionsHelper.FromCSharp9).Verify();
```

Comment:
Shouldn't it use WithLanguageVersion to be clear about the intention?

Reply:
No, because that one will use `C# 9` and `only C# 9`. While these tests are meant to run `From C# 9 onwards`